JWPlayer-SDK.podspec

```diff
@@ -15,10 +15,8 @@ Pod::Spec.new do |s|
 
   s.ios.vendored_frameworks = "JWPlayer_iOS_SDK.framework"
 
-  s.pod_target_xcconfig = {
-      'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'
-  }
-  s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
+  s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator14.0]' => 'arm64' }
```

Comment:
can you check to see if "14.*" is allowed, so we can capture all ios 14 versions?

Reply:
Good call, I addreess the change in order to capture all iOS 14 versions.